crates/macro-support/src/parser.rs

```diff
@@ -19,29 +19,59 @@ use crate::ClassMarker;
 
 thread_local!(static ATTRS: AttributeParseState = Default::default());
 
-/// Javascript keywords which are not keywords in Rust.
-const JS_KEYWORDS: [&str; 20] = [
-    "class",
+/// Javascript keywords.
+///
+/// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#reserved_words
+const JS_KEYWORDS: [&str; 44] = [
+    "arguments", // reserved in strict mode, which we are in
```

Comment:
```suggestion
    "arguments",
```
We don't need this comment everywhere, just leave a note on the top-level comment.